sdk/program/src/borsh_utils.rs

```diff
@@ -0,0 +1,55 @@
+//! Borsh utils
```

Comment:
nit: let's just call this mod `borsh` instead of `borsh_utils`.  The utils suffix is superfluous and makes you have to remember if it's "utils" or "util"